package.json

```diff
@@ -42,7 +42,8 @@
     "test:prod:watch": "MODE=prod wtr --watch",
     "bundle": "rollup -c rollup.config.js",
     "bundle:clean": "rimraf dist",
-    "bundle:watch": "rollup -c -w"
+    "bundle:watch": "rollup -c -w",
+    "prepare": "husky install"
```

Comment:
This requires developers to opt-in to the husky git hook, and it's not a descriptive script name; prepare what?  Instead let's have the hook enabled by default using the npm `postinstall` script: https://typicode.github.io/husky/getting-started.html#install-1

Reply:
Are we planning to make this a public package on npm at some point?

It's relevant for `postinstall` as it will fail when used in a public package.

```
{
  "private": true, // ← your package is private, you only need postinstall
  "scripts": {
    "postinstall": "husky install"
  }
}
```
vs
```
{
  "private": false, // ← your package is public
  "scripts": {
    "postinstall": "husky install",
    "prepack": "pinst --disable",
    "postpack": "pinst --enable"
  }
}
```

Reply:
The package is [already published on NPM](https://www.npmjs.com/package/@legumeinfo/web-components), so yes it's a public package. Can you provide a documentation link for this `postinstall` behavior? We don't actually want the post install script to run when people install the package via NPM since it's a dev script.

Reply:
yea no problem.

https://typicode.github.io/husky/getting-started.html#yarn-2

Step 3. Read the "Tip" section in green.